draft-ietf-acme-acme.md

```diff
@@ -395,8 +402,8 @@ An ACME server MUST implement the "ES256" signature algorithm {{!RFC7518}} and S
 The "jwk" and "kid" fields are mutually exclusive. Servers MUST reject requests
 that contain both.
 
-For newAccount requests, and for revokeCert requests authenticated by certificate
-key, there MUST be a "jwk" field. This field MUST contain the public key
+For newAccount requests, and for revokeCert requests authenticated by a certificate
+keys, there MUST be a "jwk" field. This field MUST contain the public key
```

Comment:
@seanturner I think "keys" should be singular now: "a certificate key". Sorry to flag one last thing.